crdppf/extract.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, request):
         # document id
         self.id = datetime.now().strftime("%Y%m%d%H%M%S")
         # sets the creation date of the PDF instance
-        self.creationdate = datetime.now().strftime("%d.%m.%Y %H:%M:%S")
+        self.creationdate = datetime.now().strftime("%d.%m.%Y   %H:%M:%S")

```

Comment:
Sure about these multiple spaces, right ?

Reply:
yeah, unfortunately - it's to meet the formatting of the pdf footer 🙈